qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -49,8 +49,10 @@
         gagent_download_url = "https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/latest-qemu-ga/${qemu_ga_pkg} -O ${gagent_host_path}"
         gagent_guest_dir = "C:\qemu-ga"
         src_qgarpm_path= "qemu-ga-win*.rpm"
+        src_qgarpm_path_new= "mingw-qemu-ga-win*.rpm"
```

Comment:
Suggest to add an annotation here to point out ***_new is used for the new qga version.What do you think?